lxd/container_lxc.go

```diff
@@ -6375,6 +6400,135 @@ func (c *containerLXC) removeUnixDevices() error {
 	return nil
 }
 
+func (c *containerLXC) insertProxyDevice(devName string, m types.Device) error {
+	if !c.IsRunning() {
+		return fmt.Errorf("Can't add proxy device to stopped container")
+	}
+
+	proxyValues, err := setupProxyProcInfo(c, m)
+	if err != nil {
+		return err
+	}
+
+	proxyPid, err := shared.RunCommandGetPid(
+		c.state.OS.ExecPath,
+		"proxy_dev_start",
+		proxyValues.listenPid,
+		proxyValues.listenAddr,
+		proxyValues.connectPid,
+		proxyValues.connectAddr,
+		"0")
+	if err != nil {
+		return fmt.Errorf("Error occurred when starting proxy device: %s", err)
+	}
+
+	err = createProxyDevInfoFile(c.DevicesPath(), devName, proxyPid)
+	if err != nil {
+		process, _ := os.FindProcess(proxyPid)
+		process.Kill()
```

Comment:
syscall.Kill(proxyPid, syscall.SIGKILL) would be more consistent with what we do elsewhere in the code

Reply:
Actually, SIGTERM in this case I guess